tutorials/cosmwasm.md

```diff
@@ -145,6 +145,10 @@ And build the binary:
 make install
 ```
 
+::: tip
+You will have to install `gcc` if you are trying it on a clean linux vm.
```

Comment:
I would move this to before the wasm installation steps and make it a proper install requirement with links/steps on how to install.

Reply:
Thought the same, but realised that it's more rare to not have gcc installed than installed. So decided to make it as a tip.

Reply:
Plus it clearly states that gcc is missing, and it's no brainer to install it.